awx/settings/defaults.py

```diff
@@ -292,6 +293,20 @@ def IS_TESTING(argv=None):
     'solo'
 )
 
+###############################################################################
```

Comment:
will make new settings file and kick this over there

Reply:
@AlanCoding low priority, but I was poking at this yesterday and got the uwsgi thing worked out, but I'm confused about where it is declared what settings file should be used. If you can point out where I end up saying, "hey, use this settings file", that would help me out.